src/views/states/list/hooks/useInterfaceColumns.ts

```diff
@@ -0,0 +1,53 @@
+import { NodeNetworkStateModelRef } from 'src/console-models';
+import { useNMStateTranslation } from 'src/utils/hooks/useNMStateTranslation';
+
+import {
+  K8sResourceCommon,
+  TableColumn,
+  useActiveColumns,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+const useInterfaceColumns = (): [
+  TableColumn<K8sResourceCommon>[],
+  TableColumn<K8sResourceCommon>[],
+] => {
+  const { t } = useNMStateTranslation();
+
+  const columns: TableColumn<K8sResourceCommon>[] = [
+    {
+      title: t('Name'),
+      id: 'name',
+      props: { className: 'pf-m-width-20' },
+    },
+    {
+      title: t('IP address'),
+      id: 'ip-address',
+    },
+    {
+      title: t('Ports'),
+      id: 'ports',
+    },
+    {
+      title: t('Mac address'),
+      id: 'mac-address',
+    },
+    {
+      title: t('LLDP'),
+      id: 'lldp',
+    },
+    {
+      title: t('MTU'),
+      id: 'mtu',
+    },
+  ];
+
+  const [activeColumns] = useActiveColumns<K8sResourceCommon>({
+    columns: columns,
+    showNamespaceOverride: false,
+    columnManagementID: `${NodeNetworkStateModelRef}-interfaces`,
+  });
+
+  return [columns, activeColumns];
```

Comment:
If you return an object instead of an array here you can pull only what you want out of it without using '_' as a placeholder.

Reply:
thanks :-)